src/m365/entra/commands/pim/pim-role-request-list.ts

```diff
@@ -0,0 +1,186 @@
+import { UnifiedRoleAssignmentScheduleRequest } from '@microsoft/microsoft-graph-types';
+import GlobalOptions from '../../../../GlobalOptions.js';
+import { Logger } from '../../../../cli/Logger.js';
+import GraphCommand from '../../../base/GraphCommand.js';
+import commands from '../../commands.js';
+import { validation } from '../../../../utils/validation.js';
+import { entraUser } from '../../../../utils/entraUser.js';
+import { entraGroup } from '../../../../utils/entraGroup.js';
+import { odata } from '../../../../utils/odata.js';
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  userId?: string;
+  userName?: string;
+  groupId?: string;
+  groupName?: string;
+  createdDateTime?: string;
+  status?: string;
+  includePrincipalDetails?: boolean;
+}
+
+class EntraPimRoleRequestListCommand extends GraphCommand {
+  private readonly allowedStatuses = ['Canceled', 'Denied', 'Failed', 'Granted', 'PendingAdminDecision', 'PendingApproval', 'PendingProvisioning', 'PendingScheduleCreation', 'Provisioned', 'Revoked', 'ScheduleCreated'];
+  public get name(): string {
+    return commands.PIM_ROLE_REQUEST_LIST;
+  }
+
+  public get description(): string {
+    return 'Retrieves a list of PIM requests for roles';
+  }
+
+  constructor() {
```

Comment:
Seems like you forgot defaultProperties that were specified. Let's add these as this is a list command. Also don't forget to update the doc file.